docker/README.md

```diff
@@ -0,0 +1,47 @@
+You can use Docker to easily try out or test sendgrid-python.
+
+<a name="Quickstart"></a>
+# Quickstart
+
+1. Install Docker on your machine
```

Comment:
I am trying to build the Dockerfile locally and I am getting a strange error:
```
→ docker build /path/to/my/dir/samundra/docker/Dockerfile
unable to prepare context: context must be a directory: /path/to/my/dir/samundra/docker/Dockerfile
```

Can you please add instructions about how to build this as step 2? 

We will eventually update this when we can get it into our dockerhub!

Reply:
I am also curious if you would be interested in adding a docker-compose.yml file, which 
would allow local volumes and other configurations to be dockerized :)

Reply:
Hi @mbernier 

Thanks for comment. 

```
docker build /path/to/my/dir/samundra/docker/Dockerfile
```
The error is because the instructions were directly copied and pasted. Here path to Dockerfile should be replaced with the one you have in local.

I will add more instructions to how to build the docker image and use it to create containers based on that image. Since I was expecting it to get to dockerhub, I didn't write this instruction strictly assuming whoever is going to use it would be familiar with these instructions.


> Can you please add instructions about how to build this as step 2?
Sure, I will add the instruction to build and setup it.

> I am also curious if you would be interested in adding a docker-compose.yml file
That would be great indeed. I can add ```docker-compose.yml``` and setup the complete working dev environment. I have found some limitations on windows since the volumes are not mounted properly.

- I will update this PR sometime today/tomorrow with the changes.
- I will also update the usage instructions.

Thanks,